tsconfig.json

```diff
@@ -83,6 +83,6 @@
       ]
     },
     "transpileOnly": true,
-    "compiler": "ttypescript"
+    "compiler": "typescript"
```

Comment:
@Jack-Works

Reply:
This is not a typo. ttypescript is a library.